src/Button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Button: React.FC<React.ButtonHTMLAttributes<HTMLButtonElement>> = props => {
+  const { children, ...remainingProps } = props;
```

Comment:
Why did you use the rest operator to save the remaining props into 'remainingProps' instead of destructuring them out?